python-regression/tests/features/steps/local_snapshots_steps.py

```diff
@@ -75,3 +78,100 @@ def read_ls_metadata(step, node):
+        "Expected to export {} addresses, instead exported {}".format(expected_num_exports, num_exports)
+    assert checksum == expected_checksum, \
+        "The checksum {} does not match the expected: {}".format(checksum, expected_checksum)
+
+    logger.info("Spent addresses file exported as: {}".format(request_return['ixi']['fileName']))
+
+    try:
+        podname = world.machine['nodes'][node]['podname']
+        config.load_kube_config()
+        v1 = client.CoreV1Api()
+        response = stream(v1.connect_get_namespaced_pod_exec,
+                          podname, 'default',
+                          command=['cat', '/iri/data/spentAddresses.txt'],
+                          stderr=True, stdin=True, stdout=True, tty=False,
+                          _preload_content=False)
+
+        while response.is_open():
+            response.update(timeout=1)
+            if response.peek_stdout():
+                logger.info("Current directory: " + os.getcwd())
+                logger.info(os.listdir(os.getcwd()))
+                f = open("/tmp/spentAddresses.txt", "w+")
+                f.write(response.read_stdout())
+
+    except:
+        logger.info("No pod name found, skipping file transfer from k8s pod")
+
+
+@step(r'the spent addresses are imported on "([^"]+)" from:')
+def merge_spent_addresses_file(step, node):
+    """
+    Reads the spentAddresses.txt file using the specified path and file name in the Gherkin options, and merges it into
+    the provided node. Then the return values for the IXI call are checked to make sure no failures occurred.
+
+    :param node: The node that the spent addresses will be merged into
+    """
+    args = step.hashes
+    for x in range(len(args)):
+        key = args[x]['keys']
+        value = args[x]['values']
+
+        if key == 'basePath':
+            base_path = value
+        elif key == 'file':
+            file_name = value
+
+    spent_addresses_file = base_path + file_name
+    logger.info(spent_addresses_file)
+    command = {"command": "merge-spent.mergeSpentAddresses", "files": [spent_addresses_file]}
+    request_return = api_utils.send_ixi_request(node, command)
+    assert 'ixi' in request_return, "Error: {}".format(request_return['error'])
+
+    failures = request_return['ixi']['errors']
+    assert len(failures) == 0, "There were failures in the merge request: {}".format(failures)
+
+
+@step(r'reading the exported spent addresses file on "([^"]+)" returns with:')
+def read_spent_addresses_file(step, node):
+    """
+    Reads the spentAddresses.txt file that should be present on the node, and ensures the expected values are there.
+
+    :param step.hashes: The pointer to the expected values that should be present in the file
+    :param node: The node whose exported spent addresses file will be read
+    """
+    arg_list = step.hashes
+
+    options = {}
+    api_utils.prepare_options(arg_list, options)
+
+    file_name = '/tmp/spentAddresses.txt'
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Probable insecure usage of temp file/directory.](https://app.codacy.com/manual/GalRogozinski/iri/pullRequest?prid=3955655)